src/v3/src/components/Form/Form.tsx

```diff
@@ -94,6 +96,7 @@ const Form: FunctionComponent<{
       data-se="o-form"
       style={{ maxWidth: '100%', wordBreak: 'break-word' }}
     >
+      <InfoSection message={message} />
```

Comment:
Moved the `InfoSection` component here so client side errors will be inside the `form` element and match V2.

Reply:
See question on Widget component about passing through the `message` prop

Reply:
ah nvm. it's coming from `useWidgetContext`